cli/README.md

```diff
@@ -65,6 +65,12 @@ To upload an algorithm, use the following command:
 ./build/cocos-cli agent algo /path/to/algorithm <private_key_file_path>
 ```
 
+##### Flags
+- -a, --algorithm string        Algorithm type to run (default "bin")
```

Comment:
Do we need this since the cli is self documenting?

Reply:
yes we have for other files, for a quick readthrough